src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -106,7 +110,7 @@ public static void main(String[] args) throws Exception {
 
 		// S3 properties
 		bucketName = cmd.getOptionValue("bucket-name");
-		awsRegion = cmd.getOptionValue("region", "us-east-1");
+		awsRegion = cmd.getOptionValue("region", "us-east-2");
```

Comment:
I'm not sure where this region should be hosted, but the existing property is working for the current implementation. We'll likely want to leave it as-is unless AWS has changed it to be the us-east-2

Reply:
This change has been reverted.